cni/network/network.go

```diff
@@ -349,6 +372,23 @@ func (plugin *netPlugin) Add(args *cniSkel.CmdArgs) error {
 	iptables.DisableIPTableLock = nwCfg.DisableIPTableLock
 	plugin.setCNIReportDetails(nwCfg, CNI_ADD, "")
 
+	// acquire cni lock file
+	// TODO: check if we need pluginconfig and if not remove it
+	config := &common.PluginConfig{Store: plugin.Store}
+	if err := acquireLockForStore(config, plugin); err != nil {
+		log.Errorf("Couldn't acquire lock: %+v", err)
+		return err
+	}
+
+	defer releaseLockForStore(plugin)
+
+	// restore network state
+	err = plugin.nm.Initialize(config, rehydrateNetworkInfoOnReboot)
+	if err != nil {
+		log.Printf("[cni-net] Failed to initialize network manager, err:%v.", err)
```

Comment:
for readability i would like to see space in `err:%v.` -> `err: %v.`, but that applies to the whole codebase so i won't block for that